src/Support/HigherOrderMessage.php

```diff
@@ -88,9 +102,16 @@ public function call(object $target)
         }
     }
 
+    public function when(callable $condition): self
```

Comment:
Can you type the format of the callable?

Reply:
Updated 👍